### PR TITLE
fix(worker): shift ipfs to the host scope

### PR DESCRIPTION
**Pull request by gurinderu**

### CODE REVIEW

File: fluence.yaml [hunk 1]
```diff
@@ -4,17 +4,17 @@
...
-    "@fluencelabs/installation-spell": "../../src/aqua/installation-spell"
-    "@fluencelabs/aqua-lib": 0.9.0
-    "@fluencelabs/registry": 0.9.2
-    "@fluencelabs/aqua-ipfs": 0.5.28
-  cargo:
-    marine: 0.14.0
-    mrepl: 0.21.3
+aquaDependencies:
+  "@fluencelabs/spell":  file:./src/aqua/spell/fluencelabs-spell-0.6.9.tgz
+  "@fluencelabs/installation-spell": file:./src/aqua/installation-spell/fluencelabs-installation-spell-0.6.9.tgz
```
Comment (justprosh) [inline fluence.yaml:13]: what is this tgz???

Comment (gurinderu) [reply fluence.yaml:13]: it is a magic from the cli team.

Comment (justprosh) [reply fluence.yaml:13]: hmmmm, okay, i don't know anything about it

**Review by justprosh — APPROVED** [review]

File: fluence.yaml (hunk 1 above)
Comment (gurinderu) [reply fluence.yaml:13]: in fact it is a result of the calling `pnpm pack`

Comment (justprosh) [reply fluence.yaml:13]: what is the problem with the previous one?

Comment (gurinderu) [reply fluence.yaml:13]: idk, it is a result of the automigration + some manual fixes

**Review by kmd-fl — APPROVED** [review]

**Pull request merged**